ui/app/templates/components/fs-directory-entry.hbs

```diff
@@ -0,0 +1,15 @@
+<tr data-test-entry>
+  <td>
+    {{#link-to "allocations.allocation.task.fs" task.allocation task pathToEntry activeClass="is-active"}}
+      {{#if entry.IsDir}}
+        <span data-test-directory-icon>{{inline-svg "folder-outline" class="icon"}}</span>
+      {{else}}
+        <span data-test-file-icon>{{inline-svg "file-outline" class="icon"}}</span>
+      {{/if}}
+
+      <span class="name" data-test-name>{{entry.Name}}</span>
+    {{/link-to}}
+  </td>
+  <td class="has-text-right" data-test-size>{{#unless entry.IsDir}}{{format-bytes entry.Size}}{{/unless}}</td>
+  <td class="has-text-right" title={{entry.ModTime}} data-test-last-modified>{{moment-from entry.ModTime interval=1000}}</td>
```

Comment:
Can you use the `format-ts` helper here to make sure that our absolute timestamp formatting is consistent?

```hbs
title={{format-ts entry.ModTime}}
```